pubsubx/message.go

```diff
@@ -0,0 +1,19 @@
+// Package pubsubx provides common utilities and formats for working with pubsub systems
+package pubsubx
+
+import (
+	"time"
+)
+
+// Message contains the data structure expected to be received when picking
+// an event from a message queue
+type Message struct {
+	SubjectURN            string                 `json:"subject_urn"`
```

Comment:
it might be nice to comment these fields since especially for usability since this is a public repo